### DISCUSSION TITLE
How to install older versions of react-native-modal?

### DISCUSSION THREAD

**Discussion by MykytaPereverziev** [react-native-modal/react-native-modal#603] [Q&A] [answered]
Tell, please, how install previous version react-native-modal? In my project I used   "react-native": "0.64.2". Thanks a lot!

**Comment by mmazzarolo** [comment, marked as answer]
Hey @MykytaPereverziev, you can install older versions of `react-native-modal` with:
```bash
# Using yarn
yarn add react-native-modal@12.1.0

# Using npm
npm i react-native-modal@12.1.0
```

For a list of available versions, see https://github.com/react-native-modal/react-native-modal/releases

**Reply by MykytaPereverziev** [reply]
Thanks! Great, it's worked for me! (I try to enter like v12, but it didn't work)
